fabric-item-api-v1/build.gradle

```diff
@@ -0,0 +1,6 @@
+archivesBaseName = "fabric-item-api-v1"
```

Comment:
Not sure if we really need a whole module just for `ItemTooltipCallback`

Reply:
To me, it makes sense. It gives a home for future item hooks. Better than a huge `misc` module that would become massively bloated as time goes on.

Reply:
Like prospector said, this is a thought of, what would likely exist in the future where this one can live